frontend/src/forms/CreateAccountFormDialog.schema.js

```diff
@@ -0,0 +1,13 @@
+import Joi from '@hapi/joi';
+
+const createAccountSchema = Joi.object({
+	firstName: Joi.string().required().label('First Name'),
+	lastName: Joi.string().required().label('Last Name'),
+	email: Joi.string()
+		.email({ minDomainSegments: 2, tlds: { allow: ['com', 'net'] } })
```

Comment:
Would it be possible to use regex to just accept all emails that end in two or more letters? 

Like this: `[a-zA-Z]{2,}`

Reply:
Had to follow JOI docs, but changed it so we (should) accept all emails.